tools/Dashboard/DevHome.Dashboard/Views/DashboardView.xaml.cs

```diff
@@ -119,6 +119,17 @@ private async Task OnLoadedAsync()
     private void OnUnloaded()
     {
         Application.Current.GetService<IAdaptiveCardRenderingService>().RendererUpdated -= HandleRendererUpdated;
+
+        Log.Logger()?.ReportDebug("DashboardView", $"Leaving Dashboard, deactivating widgets.");
+        UnsubscribeFromWidgets();
+    }
+
+    private void UnsubscribeFromWidgets()

```

Comment:
Should this be `async`?  `UnsubscribeFromWidgets` is running on the UI thread, ya?

Reply:
If this weren't in the View but rather the ViewModel, would it maybe not be running on the UI thread and we wouldn't need to Task.Run it? (At some point I want to move a bunch of stuff from View to VM)